practice/src/scripts/views/templates/tableHeader.ts

```diff
@@ -0,0 +1,52 @@
+import { getTagTemplate } from "./tags";
+
+type Attributes = {
+  "data-field": string;
+  "data-sort-label":  string;
+};
+
+type Key = 'Product name' | 'Category' | 'SKU' | 'Quantity' | 'Cost' | 'Price' | 'Status';``
```

Comment:
You are using a lot of 'key'. Please choose the specific and meaningful name for type